common/cmake/file_targets.cmake

```diff
@@ -223,7 +224,7 @@ function(ADD_FILE_TARGET)
   #
   # GENERATED must be passed if the source file is generated and is placed
   # within the binary directory.
-  set(options GENERATED)
+  set(options GENERATED IGNORE_INCLUDES)
```

Comment:
Why would IGNORE_INCLUDES ever be valid?

Reply:
The ignores are not valid until after ntemplate substitution has occurred. We **still** need a way to get which dependencies exist so we can perform ntemplate substitution on them as the current system doesn't let us rescan the dependencies after the ntemplate run has occurred.

Reply:
> The ignores are not valid until after ntemplate substitution has occurred.

But this doesn't have to be true, and Karol's was such that the includes were valid before and after.  That avoided this additional complexity, etc etc.

Reply:
Removed these changes for now but the ignore include changes where needed for both this and changes in how muxgen stuff worked in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/703, so it will likely back soon.